.github/workflows/release-operator.yml

```diff
@@ -520,7 +520,7 @@ jobs:
         run: make bundle IMG=${{ env.OPERATOR_IMAGE_REPOSITORY }}:${{ env.OPERATOR_VERSION }} VERSION=${{ env.BUNDLE_VERSION }} DEFAULT_CHANNEL=${{ env.DEFAULT_BUNDLE_CHANNEL }}
 
       - name: Process Bundle for Disconnected Support
-        uses: redhat-cop/github-actions/disconnected-csv@master
+        uses: redhat-cop/github-actions/disconnected-csv@main
```

Comment:
suggest pinning this to an SHA, `@11f2ce27643eb7c76ac3623cb99d9b08be30d762 # v4`
- https://github.com/redhat-cop/github-actions/tree/v4

Reply:
done